cvmfs/cvmfs_config

```diff
@@ -90,6 +90,7 @@ cvmfs_config_usage() {
  echo "  umount"
  echo "  wipecache"
  echo "  bugreport"
+ echo "  regularcheck"
```

Comment:
After thinking about it again, it would be better to call it `fsck` instead of `regularcheck`